packages/teams-js/src/public/constants.ts

```diff
@@ -201,3 +201,13 @@ export const errorInvalidCount = new Error('Invalid input count: Must supply a v
 export const errorInvalidResponse = new Error(
   'Invalid response: Received more images than the specified max limit in the response.',
 );
+
+/**
+ * @hidden
+ * Error codes that can be thrown from externalAppCommands and externalAppCardCommands Action Submit specifically
```

Comment:
oh, better idea, you can add it to existing, private/interfaces.ts file.

Reply:
Thanks! That makes sense, updated with a new constants.ts file under the private folder!

Reply:
@lakhveerk that's also a great idea. I have updated it with a new constants.ts file in the private folder for now. Because everything in the private/interface.ts file are interfaces. It would be great to keep this file following the interfaces pattern. What do you think?  I'm open-minded about both approaches!